src/odemis/gui/cont/features.py

```diff
@@ -87,9 +88,29 @@ def _on_btn_go_to_feature(self, _):
         if not feature:
             return
         pos = feature.pos.value
+
+        # get current position
+        pm = self._tab_data_model.main.posture_manager
+        current_label = pm.getCurrentPostureLabel()
+        logging.debug(f"Current posture: {POSITION_NAMES[current_label]}")
+
+        if current_label != FM_IMAGING: # TODO: @patrick remove this once SEM move is supported 
+            self._display_go_to_feature_warning()
+            logging.warning(f"Currently under {POSITION_NAMES[current_label]}, moving to feature position is not yet supported.")
```

Comment:
Please move the warning to above the dialog box.
Also lower its level to "info". Warning level is for potential issues detected that we don't handle. It will turn the log button to orange to attract the attention of the user. Now that the move is stopped, it's not that bad, and the user is fully aware already, so "info" is enough.